tests/test_pixels2Coo.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python3
+#
+# Testing for the helper function readMcool in hicrep/utils
+#
+import pytest
+from hicrep.utils import (
+    readMcool, cool2pixels, pixels2Coo, getSubCoo
+    )
+
+def testPixels2Coo():
+    fmcool = "tests/data/human_hi-c/4DNFICQK4N8B.mcool"
+    cool, binSize = readMcool(fmcool, 25000)
+
+    #cool2pixels gets implicitly tested here
+    pix = cool2pixels(cool)[:]
+    bins = cool.bins()[:]
+    sparse = pixels2Coo(pix, bins)
```

Comment:
Can you rename `sparse` to something like `mSparse` to avoid potential name collision with `scipy.sparse`?

Reply:
Done